runtests.py

```diff
@@ -9,12 +9,20 @@
 
 if __name__ == '__main__':
     from django.core.management import execute_from_command_line
+    from django.conf import settings
 
     args = sys.argv
     args.insert(1, 'test')
     if not os.environ.get('SAMPLE_APP', False):
+        # Temporarily added to identify slow tests
+        setattr(
+            settings,
+            'TEST_RUNNER',
+            'openwisp_monitoring.measure_speed.TimeLoggingTestRunner',
```

Comment:
why is it only enabled in this part of the if and not in the other?

BTW, maybe we can put this in the settings.py of the test project, it should be easier to find it, here is kind of hidden and can generate confusion.